data/pages-content/sponsor/information.md

```diff
@@ -56,126 +53,130 @@ For details, please refer to the relevant sections below.
 - **Assign the tickets** to every team member that you have chosen to attend the
   conference. You can assign them by changing your order details after you make
   the order. Click the order URL in your order confirmation email from
-  [support@pretix.eu](mailto:support@pretix.eu) with the **Subject: Your
+  [support@pretix.eu](mailto:support@pretix.eu) with the Subject: **Your
   order**: **_XXXX_** and amend accordingly.
 
-**_Deadline_** for registering and assigning the tickets: **_June 27_**
+**Deadline** for registering and assigning the tickets: **19 June, 2023**
 
 </details>
 
----
 
 # Exhibition
 
+
 <details>
-  <summary><h2>Exhibition Schedule</h2></summary>
+  <summary><h3>Exhibition Schedule</h3></summary>
```

Comment:
Could we try smaller headers? h3 looks huge on my screen

Reply:
used h4 in https://github.com/EuroPython/website/pull/259/commits/346c8a8083779a7fd961e689b3f14e18b1f772e3 Does it look better?

Reply:
I like this more 🤗